cmdeploy/src/cmdeploy/cmdeploy.py

```diff
@@ -345,6 +367,12 @@ def get_sshexec():
     args.get_sshexec = get_sshexec
 
     out = Out()
+    if is_pytest(): ## issue: https://github.com/chatmail/relay/issues/622
```

Comment:
not a good idea to make production-used code test whether it's in a test environment because that can easily introduce bugs that only occur in production.